py/torch_tensorrt/dynamo/conversion/aten_ops_converters.py

```diff
@@ -2133,6 +2133,27 @@ def aten_ops_linear(
     )
 
 
+@dynamo_tensorrt_converter(torch.ops.aten._cdist_forward.default)
+@dynamo_tensorrt_converter(torch.ops.aten._cdist_forward)
```

Comment:
Typically, `torch.ops.aten._cdist_forward` is equal to `torch.ops.aten._cdist_forward.default`. (except in some rare cases) If deleting `torch.ops.aten._cdist_forward` works, then please delete it.

Reply:
I have deleted `@dynamo_tensorrt_converter(torch.ops.aten._cdist_forward)` and verified that it works.